third_party/mavlink/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ endif()
 ExternalProject_add(
     mavlink
     GIT_REPOSITORY https://github.com/mavlink/mavlink
-    GIT_TAG 0b5b5dabf9c6ae22f2f93f0db9a4e19dcf6f19ac
```

Comment:
I think I disagree with this. The effect will be that it will randomly break the build for users, who will at least be frustrated and hopefully will open an issue, which will put pressure on the maintainers (because "fix it ASAP it's broken now!").

Also ideally that should not be needed without adding a new feature to MAVSDK unless MAVLink broke backward compatibility, right? And if the problem is that MAVLink keeps breaking backward compatibility, I would say this is a MAVLink problem :innocent:.

Reply:
Alright, fair enough. I will revert this back to the "latest" commit hash in MAVLink.

Reply:
@JonasVautherin done